codec-http2/src/test/java/io/netty/handler/codec/http2/HpackTest.java

```diff
@@ -67,8 +57,9 @@ public HpackTest(String fileName) {
         return data;
     }
 
-    @Test
-    public void test() throws Exception {
+    @ParameterizedTest(name = "file = {0}")
+    @MethodSource("data")
+    public void test(String fileName) throws Exception {
```

Comment:
I think it's no longer necessary for `data` to return the arguments as a collection of _arrays_. It should be possible to just be a collection/stream/list/array of strings. For instance, the `File[]` array could be returned directly if the test took `File` as its argument.

Reply:
TIL... Let me refactor